### DISCUSSION TITLE
Copy Document as Image

### DISCUSSION THREAD

**Discussion by YaLTeR** [flxzt/rnote#752] [Ideas]
I sometimes use rnote as a screenshot annotation tool: I'll take a screenshot, paste it into rnote, then draw something on top. The missing piece seems to be the ability to copy the result back to clipboard as an image; I usually resort to just taking a screenshot of the result again, but this is sometimes awkward, especially if the original screenshot was already full-screen.

Would it make sense for rnote to have a "copy document as image" menu entry/hotkey that will take the non-empty bounding box and copy it as a PNG at 100% scale to clipboard?

**Comment by flxzt** [comment]
This should already be possible by selecting the content and then copying into the clipboard via `Ctrl-C` or the `Clipboard->Copy` menu entry. The clipboard should then contain the selection as `Svg` and `Png`. You can also export the selection with some options through the "Export Selection" menu/dialog.
Next release will also add a export preview feature which got merged very recently (#734)

This should cover your use case, right?

**Reply by YaLTeR** [reply]
Oh. Huh. I guess I tried Ctrl-A to select everything, it did nothing, then I tried the Copy menu entry with no selection, it also did nothing, so I figured that it wasn't supported. Manually zooming out and rectangle-selecting everything and copying does what I want, yeah.

Could Ctrl-A be added then (unless I'm missing something else)?

**Reply by flxzt** [reply]
`Ctrl + a` to select everything is already implemented, but the keyboard focus probably was still in the bottom bar. We should look into if this can be improved without breaking keyboard navigation

**Reply by YaLTeR** [reply]
Oh yeah, indeed. I click the brush then draw something, and it seems the focus just stays on the bottom bar no matter what I do. Only clicking selection then clicking on the canvas lets me Ctrl-A it seems.

**Comment by YaLTeR** [comment]
Well, guess I can close this since the thing already exists.
